CHANGES.rst

```diff
@@ -4,7 +4,20 @@ Changes
 2.1 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Add support for SQLAlchemy 2.0.
+  (`#79 <https://github.com/zopefoundation/zope.sqlalchemy/pull/79>`_)
+
+- Add support for new psycopg v3 backend.
+  (`#79 <https://github.com/zopefoundation/zope.sqlalchemy/pull/79>`_)
+
+- No longer allow calling ``session.commit()`` within a manual nested database
```

Comment:
Could you please mark this entry as a breaking change. (I can do the version bump to 3.0 before the release.)